### PR TITLE
Adding support and instructions for mac users

### PR DESCRIPTION
**Pull request by legojoshua12**

### CODE REVIEW

File: QModManagerMac/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# How to install Mods on Mac:
+
+## Prerequisites
+
+Download and install [Mono Project](https://www.mono-project.com/download/stable/)
```
Comment (Thomazella) [inline QModManagerMac/README.md]: Suggest adding to Prerequisites that files in fork are needed:
``` markdown
Click the green button to download repo as zip. Find folder QModManagerMac inside.
```

Comment (Thomazella) [reply QModManagerMac/README.md]: would send a PR but this is already a PR lol

**Comment by Thomazella** [comment]
Tested and it works.

**Comment by AlexejheroYTB** [comment]
I changed the base branch from `master` to `1.4-dev`. Please let me rename the file handle all merge conflicts

**Comment by AlexejheroYTB** [comment]
@legojoshua12 In the meantime, please tell me what needs to be done when the QModManager project is built. Should the `QModManager.exe` and `QModInstaller.dll` files be copied over into this folder, or is there something else that needs to be done first?

**Comment by legojoshua12** [comment]
Ok, I renamed it. As for when it is built, do you mean what of those files are necessary? This folder being added essentially gives all 4 override files needed for mac installation.  When you are making new updates to QModManager, I believe you can just update the exe (but don't quote me on that, I haven't tried). The installer dll can't be changed, since it is what locates all the files.

**Comment by AlexejheroYTB** [comment]
I was asking if the files need some special treatment to work on mac before updating them in the folder. Also, the installer DLL is updated every time and needs to be updated in the folder too.

**Comment by legojoshua12** [comment]
No, they don't need any special treatment of any sort, and should work just fine when updating them.

**Comment by AlexejheroYTB** [comment]
@legojoshua12 Is this PR complete?

**Comment by legojoshua12** [comment]
Yes, it is complete and you can merge it whenever you want to.  For an installer I will make a separate PR.

**Pull request merged**